src/OmniSharp.Host/Startup.cs

```diff
@@ -189,15 +189,28 @@ public void ConfigureServices(IServiceCollection services)
                 }
                 catch (Exception e)
                 {
-                    var message = $"The project system '{projectSystem.GetType().Name}' threw exception during initialization.\n{e.Message}\n{e.StackTrace}";
+                    var message = $"The project system '{projectSystem.GetType().Name}' threw exception during initialization.";
                     // if a project system throws an unhandled exception it should not crash the entire server
-                    logger.LogError(message);
+                    logger.LogError(e, message);
+                }
+            }
+
+            // run all workspace options providers
+            foreach (var workspaceOptionsProvider in PluginHost.GetExports<IWorkspaceOptionsProvider>())
+            {
+                try
+                {
+                    Workspace.Options = workspaceOptionsProvider.Process(Workspace.Options);
```

Comment:
This implies an order to the `IWorkspaceOptionsProviders`. Is there a way we can define that?